src/setup.rs

```diff
@@ -92,7 +108,10 @@ fn setup_kani_bundle(kani_dir: &Path, use_local_bundle: Option<OsString>) -> Res
             .arg("-zxf")
             .arg(&path)
             .current_dir(&kani_dir)
-            .run()?;
+            .run()
+            .context(
+                "Failed to extract tar file, try removing Kani setup and restarting the setup step",
```

Comment:
What exactly should the user do to remove "Kani setup?"

Reply:
Can I add a direct suggestion to remove it by invoking `rm ~/.kani`?